components/CommentCount.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+class CommentCount extends React.Component {
+  formatString (count) {
+    if (count == undefined) { count = 0 }
```

Comment:
If commentCount is a required prop in the NewsItem, will `count` ever be undefined?